public/tutorials/setup-for-manual-relay.sh

```
@@ -0,0 +1,152 @@
+#! /bin/sh
+
+rm -rf manual-relay
+mkdir manual-relay
+cd manual-relay
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)